docs/markdown/FAQ.md

```diff
@@ -405,6 +405,8 @@ advantages:
    not care what the extension is](https://docs.microsoft.com/en-us/cpp/build/reference/link-input-files?view=vs-2019),
    so specifying `libfoo.a` instead of `foo.lib` does not change the workflow,
    and is an improvement since it's less ambiguous.
+1. Projects built with MinGW compiler using UCRT (e.g. MSYS2) are compatible
+   with MSVC and also name their static libraries `libfoo.a`.
```

Comment:
```suggestion
1. Projects built with MinGW compiler are fully compatible with 
   MSVC as long as they use the same CRT (e.g. UCRT with MSYS2).
   These projects also name their static libraries `libfoo.a`. 
```

Reply:
Reason for this is that previously MSYS used MSVCRT, and so did Visual Studio. The two were compatible back then too. The incompatibility began with Visual Studio 2015 when MSVC switched to UCRT and MinGW continued with MSVCRT till recently.

Reply:
Thanks for clarification, I like that phrasing. Updated.